cms/envs/common.py

```diff
@@ -50,8 +50,9 @@
 # Although this module itself may not use these imported variables, other dependent modules may.
 from lms.envs.common import (
     USE_TZ, TECH_SUPPORT_EMAIL, PLATFORM_NAME, PLATFORM_DESCRIPTION, BUGS_EMAIL, DOC_STORE_CONFIG, DATA_DIR,
-    ALL_LANGUAGES, WIKI_ENABLED, update_module_store_settings, ASSET_IGNORE_REGEX,
-    PARENTAL_CONSENT_AGE_LIMIT, REGISTRATION_EMAIL_PATTERNS_ALLOWED,
+    STATIC_ROOT_BASE, LOG_DIR, LOCAL_LOGLEVEL, CERT_QUEUE, PASSWORD_POLICY_COMPLIANCE_ROLLOUT_CONFIG,
```

Comment:
Can you make copies of these instead of importing them?

Since our plan is to eventually separate these services (LMS and CMS)
I would like to decouple them as much as possible.